modules/ob-disabling-shipwright-custom-resource.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-a-shipwright-build-resource_{context}"]
+= Disabling ShipwrightBuild resource
+
+You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources CSI driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
+
+
+.Procedure
+
+* Disable the `Shipwright` CR from the `openshift-builds` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc patch openshiftbuild cluster -n openshift-builds --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
```

Comment:
Namespace not necessary - `OpenShiftBuild` is a cluster-scoped resource. See [crd definition](https://github.com/redhat-openshift-builds/operator/blob/main/config/crd/bases/operator.openshift.io_openshiftbuilds.yaml#L15)

```suggestion
$ oc patch openshiftbuild cluster --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
```